### DISCUSSION TITLE
Welcome to Spring Cloud Zuul Rate Limit Discussions!

### DISCUSSION THREAD

**Discussion by marcosbarbero** [marcosbarbero/spring-cloud-zuul-ratelimit#408] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## ❌ Reporting Problems

  * If you're facing issues while using this library, please file an [issue](../issues).

**Comment by prem5192** [comment]
@marcosbarbero 
Hey I am trying to use the spring-cloud-zuul-ratelimit-core:2.4.2.RELEASE dependency. I have mentioned the following in my yml file.
ratelimit:
    enabled: true
    repository: JPA
    behind-proxy: true
    policy-list:
      product-service: 
        - limit: 5
          refresh-interval: 60
      shipping-service:
        - limit: 5
          refresh-interval: 60
When i hit the product-service / shipping-service, the remaining column in the rate table is not reducing. It reduces only once when I hit the service the first time. The consecutive hits to the service does not reduce the limit value and the I am not receiving the TOO_MANY_REQUESTS error with the response code 429.
I am trying this for the past 2 weeks but not sure what is happening.
